src/bot.py

```diff
@@ -145,7 +145,7 @@ def dijkstra_algorithm(self, point):
         """Finds the shortest paths from the point to all other points.
 
         :param point: int - point index
-        :return: dictionary of shortest paths
+        :return: 2-tuple of dictionaries. Dictionary of shortest paths and dictionary distance of paths
```

Comment:
:return: 2-tuple of dictionaries where the first one is shortest paths and the second one is distance of paths

Might be a little bit better